doxyqml/doxyqml.py

```diff
@@ -6,9 +6,15 @@
 import re
 import sys
 
-from . import qmlparser
-from .lexer import Lexer, LexerError
-from .qmlclass import QmlClass
+# On some systems (e.g., Windows 10 with Python 3.6.5), relative imports
+# (e.g., from . import qmlparser) are not allowed. Therefore, insert the
+# path to doxyqml package in PYTHONPATH, so that imports are possible.
+doxyqmlPackage_path = os.path.join(os.path.dirname(__file__), os.pardir)
+sys.path.insert(0, doxyqmlPackage_path)
+
+import qmlparser
+from lexer import Lexer, LexerError
+from qmlclass import QmlClass
```

Comment:
Removing relative imports is a good idea, but you should not be altering `sys.path`. Yes, some of the tests are doing this, but it's a bad idea, and with what I write here we can get rid of this. The idea is to do all work on Doxyqml inside a virtualenv. This is how I would go about it:

## Setup (only done once)
1. Create a virtualenv to work on Doxyqml. I like to use [pew](https://github.com/berdario/pew) for that. So, first install pew: `pip3 install pew`, then create a virtualenv with it: `pew new doxyqml`.
2. Install Doxyqml in "development mode": cd to the root of your Doxyqml source tree and run `pip install -e .`.
The development mode makes it so that when you `import doxyqml` it imports the code inside your source tree. This means you can now use real package import for Doxyqml instead of using relative ones or altering sys.path.
3. With this in place, you can replace all relative imports with imports like this:
```
from doxyqml import qmlparser
from doxyqml.lexer import Lexer, LexerError
from doxyqml.qmlclass import QmlClass
```

## Working with the virtualenv
1. Everytime you want to work on Doxyqml, activate your virtualenv with `pew workon doxyqml`.

Reply:
I am not sure I understand. Just to make sure we are on the same page, here is my workflow:

I installed doxyqml using "python setup.py install". Then tried to run manual tests by running doxygen after editing the Doxyfile. Here is the output I got:
  Preprocessing C:/python3/Lib/site-packages/doxyqml_fork/doxyqml/tests/manual/AnonymousFunction.qml... 
 Traceback (most recent call last):
  File "C:\python3\Lib\site-packages\doxyqml\doxyqml.py", line 9, in <module>
    from . import qmlparser
  ImportError: cannot import name 'qmlparser'

I get the same error when I run doxygen from my 'real' source code. Thus, the problem I am seeing is while running doxygen in 'production mode'.

Reply:
I assume if you modify the imports to use `from doxyqml import foo` instead of `from . import foo` and then run `python setup.py install` it should work. I am going to give it a try in my Windows VM.